doc/environ.md

```diff
@@ -92,6 +92,11 @@ Whether to aggregate unreferenced epochs. `BOOL`. Default to false.
 ### `DAOS_TARGET_OVERSUBSCRIBE`
 Whether to accept target number oversubscribe for daos server. `BOOL`. Default to false.
 
+### `DAOS_POOL_CREATE_TIMEOUT`
```

Comment:
is it reasonable to consider something like DAOS_POOL_TGT_TIMEOUT that would apply the same timeout to both the MGMT_TGT_CREATE and MGMT_TGT_DESTROY RPCs?

Reply:
MGMT_TGT_DESTROYs should be reasonably quick, compared to MGMT_TGT_CREATE, no...?

Reply:
Yes it seems so (I'm testing some scenarios while investigating DAOS-5206). So let's abandon the thought for now and apply the setting just to MGMT_TGT_CREATE.